Sources/View/UITableView+Reusable.swift

```diff
@@ -40,25 +40,61 @@ public extension UITableView {
 
    - parameter indexPath: The index path specifying the location of the cell.
    - parameter cellType: The cell class to dequeue
+   - parameter autoRegister: Set to true in order to automatically register the cell class/nib if not registered
 
    - returns: A `Reusable`, `UITableViewCell` instance
 
    - note: The `cellType` parameter can generally be omitted and infered by the return type,
    except when your type is in a variable and cannot be determined at compile time.
    - seealso: `dequeueReusableCell(withIdentifier:,for:)`
    */
-  final func dequeueReusableCell<T: UITableViewCell>(for indexPath: IndexPath, cellType: T.Type = T.self) -> T
+    final func dequeueReusableCell<T: UITableViewCell>(
+        for indexPath: IndexPath,
+        cellType: T.Type = T.self,
+        autoRegister: Bool = false) -> T
     where T: Reusable {
-      guard let cell = self.dequeueReusableCell(withIdentifier: cellType.reuseIdentifier, for: indexPath) as? T else {
-        fatalError(
-          "Failed to dequeue a cell with identifier \(cellType.reuseIdentifier) matching type \(cellType.self). "
-            + "Check that the reuseIdentifier is set properly in your XIB/Storyboard "
-            + "and that you registered the cell beforehand"
-        )
-      }
+        if autoRegister {
+            if let bareCell = self.dequeueReusableCell(withIdentifier: cellType.reuseIdentifier) as? T {
+                return bareCell //Get a cell if we can
+            } else {
+                register(cellType: cellType) //Register class or nib
```

Comment:
I know I'm a little OCD but please add a space after the `//` at the very start of the comment (here and on line 58 just above)